src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -77,7 +77,7 @@ public static int createMonitoringTopicIfNotExists(String zkUrl, String topic, i
     ZkUtils zkUtils = ZkUtils.apply(zkUrl, ZK_SESSION_TIMEOUT_MS, ZK_CONNECTION_TIMEOUT_MS, JaasUtils.isZkSecurityEnabled());
     try {
       if (AdminUtils.topicExists(zkUtils, topic)) {
-        LOG.info("Monitoring topic \"" + topic + "\" already exists.");
+        LOG.debug("Monitoring topic \"" + topic + "\" already exists.");
```

Comment:
OK. If you are changing this to debug what do you think about setting ProduceRunnable.run catch Exception log to debug.  I'm getting a flood of EKG tickets from things that are probably failures on the broker side.